src/index.ts

```diff
@@ -171,3 +171,19 @@ export function memo<T, P extends keyof T>(
   }
   return result;
 }
+
+/**
+ * Validate if a value is an object
+ *
+ * @param {object} source
+ * @returns {Boolean}
+ */
+export function isPlainObject(source: Object): boolean {
+  const typeofSource: string = typeof source;
+  const sourcePrototype: string = Object.prototype.toString.call(source);
+  return (
+    typeofSource === 'object' &&
+    source !== null &&
+    sourcePrototype === '[object Object]'
+  );
+}
```

Comment:
```suggestion
}

```

We need a trailing line here.